profiles/AmolShelke2.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/95171638/153740577-e9b68c79-cf30-403a-82eb-d1ce9cf7aada.png",
+  "issueId": 1034,
+  "name": "Amol Shelke",
+  "username": "AmolShelke2"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```